articles/python/azure-sdk-sovereign-domain.md

```diff
@@ -23,28 +23,32 @@ Pre-defined sovereign cloud constants are provided by the `azure_cloud` module o
 
 To use a definition, import the appropriate constant from `msrestazure.azure_cloud` and apply it when creating client objects. 
 
-When using `DefaultAzureCredential`, as shown in the following example, you also need to use the appropriate value from `azure.identity.AzureAuthorityHosts`.
```

Comment:
`azure.identity.AzureAuthorityHosts` contains the same value as `CLOUD.endpoints.active_directory`, but is not used in the following code snippet, causing confusion.